foreshadow/smart/all.py

```diff
@@ -12,26 +12,30 @@
 import pandas as pd
```

Comment:
What's the need for the all file as opposed to the \_\_init\_\_.py file

Reply:
will do.